tests/test.sh

```diff
@@ -51,3 +55,5 @@ done
 printf -- "---------\n"
 printf "$success succeed, $failure failed, total : $total\n"
 
+# If one test failed it exits with 1 else 0
+exit $error
```

Comment:
Add this line to avoid the Travis CI build passing the test until I write the code which will validate the test.